helpers/getScreenings.js

```diff
@@ -0,0 +1,45 @@
+import { MongoClient } from "mongodb";
+
+export async function getAllScreenings() {
+
+    const client = await MongoClient.connect("mongodb+srv://Bosse:LKjRPJ2chOlxeM0E@cluster0.rpxxl.mongodb.net/movieScreening?retryWrites=true&w=majority");
+
+    const db = client.db();
+
+    const screeningsCollection = db.collection("movieScreening");
+
+    const screenings = await screeningsCollection.find({}).sort({ date: 1 }).toArray();
+
+    client.close();
+
+    return screenings;
+
+}
+
+export async function getSpecifikScreenings(name) {
+    const client = await MongoClient.connect("mongodb+srv://Bosse:LKjRPJ2chOlxeM0E@cluster0.rpxxl.mongodb.net/movieScreening?retryWrites=true&w=majority");
+
+    const db = client.db();
+
+    const screeningsCollection = db.collection("movieScreening");
+
+    const screenings = await screeningsCollection.find({ movie: name.toString() }).toArray();
```

Comment:
Using the name as an identifier like this is not ideal, for multiple reasons.

* What happens if the name contains special characters that need to be URL encoded?
* What happens if there are two movies with the same title (as has been the case multiple times through history)?
* What happens if you misspell the name, spread the URL containing the name on social media, and then go and correct the name? 

This is (some of the reasons) why we tend to use unique IDs, numeric or otherwise, to identify objects in a database.